### PR TITLE
php72-uuid 1.0.4 (new formula)

### PR DESCRIPTION
**Pull request by romainneutron**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/php72-uuid.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  homepage "https://pecl.php.net/package/uuid"
+
+  stable do
+    url "https://pecl.php.net/get/uuid-1.0.4.tgz"
+    sha256 "63079b6a62a9d43691ecbcd4eb52e5e5fe17b5a3d0f8e46e3c17ff265c06a11f"
+
+    patch do
+      # let's fix the path to uuid.h (uuid/uuid.h on linux, ossp/uuid.h on OSX)
+      # uuid_mac & uuid_time might not be available on OSX, let's add test to avoid compiling issue on these functions
+      url "https://gist.githubusercontent.com/romainneutron/fe068c297413aee565d5/raw/28d6ba0b6e902e82e71bb9a1ed768c836a8161e4/php-uuid-1.0.4"
```
Comment (javian) [inline Formula/php72-uuid.rb]: Has  there been any attempts of getting this merged upstream ?

Comment (romainneutron) [inline Formula/php72-uuid.rb]: I don't know

Comment (SMillerDev) [inline Formula/php72-uuid.rb]: As far as I can tell we could have https://github.com/Homebrew/homebrew-core/blob/master/Formula/ossp-uuid.rb make a symlink to the uuid include dir right? that would rid us of at least a part of the patch. UUID upstream might be more inclined to merge it if it were just the `#if HAVE_UUID_TIME` changes

Comment (javian) [inline Formula/php72-uuid.rb]: @remicollet would you be able to comment on this ?

**Comment by javian** [comment]
I'd still want to see a PR towards the git repo for uuid with the proposed changes.

File: Formula/php72-uuid.rb (hunk 1 above)
Comment (SMillerDev) [reply Formula/php72-uuid.rb]: patch for the functions is submitted upstream: https://bugs.php.net/bug.php?id=75759

**Comment by SMillerDev** [comment]
@BrewTestBot test this please

File: Formula/php72-uuid.rb (hunk 1 above)
Comment (javian) [reply Formula/php72-uuid.rb]: Looks like it was reported a long time ago already - https://bugs.php.net/bug.php?id=58311 . Remi asked to submit a PR to the git repo to fix this.

**Comment by MikeMcQuaid** [comment]
As [announced on the 19th of January](https://brew.sh/2018/01/19/homebrew-1.5.0/) this tap will be archived by 31st March 2018. Various PHP formulae have been [migrated to Homebrew/homebrew-core](https://github.com/Homebrew/homebrew-php/blob/master/tap_migrations.json). Thanks for your pull request anyway and sorry we couldn't include it. Please consider [maintaining this formula in your own tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap).

**Pull request closed without merging**